packages/dss-bridge/src/DomainGuest.sol

```diff
@@ -167,19 +167,15 @@ abstract contract DomainGuest {
 
     // --- MCD Support ---
 
-    /// @notice Set the global debt ceiling for the local dss
-    /// @param id An incrementing ID to prevent earlier updates from overwriting later ones
-    /// @param line The new global debt ceiling [RAD]
+    /// @notice Record changes in line and grain and update dss global debt ceiling if necessary
+    /// @param dline The change in the line [RAD]
     /// @param minted The amount of DAI minted into the remote escrow
-    function lift(uint256 id, uint256 line, uint256 minted) external hostOnly isLive {
-        // Need to ensure out of order updates do not incorrectly set the debt ceiling
-        if (id > liftId) {
-            vat.file("Line", line);
-            liftId = id;
-        }
+    function lift(int256 dline, uint256 minted) external hostOnly isLive {
+        line += dline;
```

Comment:
I guess the main disadvantage of this approach is that the `line` here can in theory be temporarily greater than intended if subsequent line updates (e.g. +5, -3, +3) get relayed out of order (e.g. +5, +3, -3). But considering that calls to DomainHost.lift() are in practice controlled by governance, it doesn't seem to be a scenario likely to occur in practice so IMO that's an acceptable trade-off.

Reply:
I would actually like to cover these cases if possible. Maybe it's an issue maybe not.

Reply:
Yeah I'd like to think hard on this during the offsite. Do we accept that governance needs to be careful to execute things in order? Do we need escape hatches in case a message does not successfully go through?